### ISSUE TITLE
Tests fail on Python 3.12 due to distutils removal

### ISSUE THREAD

**Issue by mweinelt** [matze/pkgconfig#64]
The distutils module has been removed from Python in 3.12, so the tests are now failing like this:

```pytb
============================= test session starts ==============================
platform linux -- Python 3.12.0, pytest-7.4.2, pluggy-1.3.0
rootdir: /build/source
collected 0 items / 1 error                                                    

==================================== ERRORS ====================================
______________________ ERROR collecting test_pkgconfig.py ______________________
ImportError while importing test module '/build/source/test_pkgconfig.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/xcr4d0ng894lbl9s874any12p7abg6iq-python3-3.12.0/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
test_pkgconfig.py:1: in <module>
    from distutils.core import Extension
E   ModuleNotFoundError: No module named 'distutils'
=========================== short test summary info ============================
ERROR test_pkgconfig.py
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
=============================== 1 error in 0.08s ===============================

**Comment by ZhangChuanHui** [comment]
+1

**Comment by ThomasWaldmann** [comment]
Fixed by #67.
